api/package.json

```diff
@@ -16,9 +16,9 @@
     "unzip-stream": "0.3.0"
   },
   "devDependencies": {
-    "@types/node": "14.18.33",
+    "@types/node": "16.18.11",
```

Comment:
**question (non-blocking):** Is everything supposed to be node@14 or no?

https://github.com/vercel/vercel/pull/9198/files#r1067359556

Reply:
yeah but not api since we deploy this with Node 16